app/models/export/activity_actuals_columns.rb

```diff
@@ -33,6 +33,10 @@ def last_financial_quarter
     financial_quarter_range.max
   end
 
+  def rows_for_last_financial_quarter
+    rows.each_with_object({}) { |(key, values), rows| rows[key] = values.last }
```

Comment:
just a little suggestion of the naming of the block variable:

```txt
rows.each_with_object({}) { |(key, values), rows| rows[key] = values.last }
 ^                     ^                      ^     ^
 |                     |                      |     |
rows (1)                ----------rows (2)-------------
is a method                       is a block variable
 
These two "rows" don't stomp on each other as they have a different scope but 
perhaps you can make the expression easier on the eye if you use a different block variable?

```
Something like:
```
rows.each_with_object({}) { |(key, values), obj| obj[key] = values.last }

```
perhaps?